src/wallet/wallet.cpp

```diff
@@ -3403,16 +3403,14 @@ void CWallet::GetKeyBirthTimes(interfaces::Chain::Lock& locked_chain, std::map<C
     for (const auto& entry : mapWallet) {
         // iterate over all wallet transactions...
         const CWalletTx &wtx = entry.second;
-        if (Optional<int> height = locked_chain.getBlockHeight(wtx.m_confirm.hashBlock)) {
-            // ... which are already in a block
-            for (const CTxOut &txout : wtx.tx->vout) {
-                // iterate over all their outputs
-                for (const auto &keyid : GetAffectedKeys(txout.scriptPubKey, *spk_man)) {
-                    // ... and all their affected keys
-                    std::map<CKeyID, int>::iterator rit = mapKeyFirstBlock.find(keyid);
-                    if (rit != mapKeyFirstBlock.end() && *height < rit->second)
-                        rit->second = *height;
-                }
+        // ... which are already in a block
+        for (const CTxOut& txout : wtx.tx->vout) {
+            // iterate over all their outputs
+            for (const auto& keyid : GetAffectedKeys(txout.scriptPubKey, *spk_man)) {
+                // ... and all their affected keys
+                std::map<CKeyID, int>::iterator rit = mapKeyFirstBlock.find(keyid);
+                if (rit != mapKeyFirstBlock.end() && wtx.m_confirm.block_height < rit->second)
```

Comment:
Is this a style requirement? IMO don't find code easier to read doing so.

Reply:
Yes, it is a style requirement. Same line or add `{}`.

From https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#coding-style-c
> If an `if` only has a single-statement `then`-clause, it can appear on the same line as the `if`, without braces. In every other case, braces are required, and the `then` and `else` clauses must appear correctly indented on a new line.